documentation/docs/libraries/java/how_to/get_block.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: How To Get A Block
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import GetBlock from '!!raw-loader!./../../../../../bindings/java/iota-client-java/examples/ExampleProject/src/node_api_core/GetBlock.java';
+import GetBlockRaw from '!!raw-loader!./../../../../../bindings/java/iota-client-java/examples/ExampleProject/src/node_api_core/GetBlockRaw.java';
+
+<CodeBlock className="language-java">
+  {GetBlock}
+</CodeBlock>
+
+<CodeBlock className="language-java">
+    {GetBlockRaw}
```

Comment:
Align indentation

Reply:
I did format all the doc files with the IDE formatter, is this fine now for you or should I format it differently?

Reply:
Should be fine